client/state/ui/selectors.js

```diff
@@ -139,3 +139,54 @@ export function hasSidebar( state ) {
 	}
 	return get( getSection( state ), 'secondary', true );
 }
+
+/**
+ * Returns viewport width
+ *
+ * @param  {Object}  state Global state tree
+ * @return {Number}  Viewport width in pixels
+ */
+export const getViewportWidth = ( state ) => ( state.ui.viewportWidth );
```

Comment:
Parens around the return value not needed.

Reply:
parens around the input `state` not needed :winK: